### PR TITLE
CE-2552 : Updated introduction page.

### PR DESCRIPTION
**Pull request by ankithooda**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,6 +2,168 @@
 layout: default
 title: IDERA Developer Site
 ---
+
+### Format of CURL Request
+
+The general format for the curl request in this. You can get the curl software from here - https://curl.haxx.se/
```
Comment (Neelakshi1808) [inline index.md]: **The general format for the curl request in this** should be **The general format for the curl is as follows.**

File: index.md [hunk 2]
```diff
@@ -2,6 +2,168 @@
...
+### Available APIs
+
+Given below is a brief description of all APIs available.
+
+#### Alert
+
+##### [Definitions](alerts/definitions.html)
+Alert Definitions define the conditions which are periodically checked and if the conditions are met then an Alert is generated.
+The following actions can be performed using this API.
+ - Index - Fetch all your Alert Definition
```
Comment (Neelakshi1808) [inline index.md]: Full stop at end (either have it every where or remove it)

File: index.md [hunk 3]
```diff
@@ -2,6 +2,168 @@
...
+Tags are used to group multiple custom metrics monitored by CopperEgg.
+The following actions can be performed using this API.
+ - Index - Fetch all defined Custom Metrics tags.
+ - Show - Fetch custom metric object associated with single custom metric tag.
+ - Create - Add a new Custom Metrics Tag to a custom metric object.
+ - Delete - Remove a Custom Metrics Tag from a custom metric object.
+ 
+ 
+### Releases
+
```
Comment (Neelakshi1808) [inline index.md]: Update the release date as well.

File: index.md (hunk 1 above)
Comment (Neelakshi1808) [inline index.md]: Check once on grammarly.

File: index.md (hunk 3 above)
Comment (ankithooda) [inline index.md]: done

File: index.md (hunk 2 above)
Comment (ankithooda) [inline index.md]: done

File: index.md (hunk 1 above)
Comment (ankithooda) [inline index.md]: done

**Pull request merged**